treeherder/intermittents_commenter/commenter.py

```diff
@@ -169,6 +169,10 @@ def print_or_submit_changes(self, all_bug_changes):
                 # sleep between comment submissions to avoid overwhelming servers
                 time.sleep(1)
 
+        if settings.COMMENTER_API_KEY is None:
```

Comment:
This logging statement will be useful long-term, even when the API key is set - so let's remove the conditional